<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_header">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>13</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>Генератор Паролів</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
     </font>
    </property>
    <property name="text">
     <string>Тут буде результат</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch8_number">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>106</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
     </font>
    </property>
    <property name="text">
     <string>Використовувати Числа</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch8_alph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
     </font>
    </property>
    <property name="text">
     <string>Використовувати алфавіт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_generator">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #000000;
border-radius: 10px;
background-color: #96e1ff</string>
    </property>
    <property name="text">
     <string>Сгенерувати</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
